packages/itwin/property-grid/src/test/components/MultiElementPropertyGrid.test.tsx

```diff
@@ -0,0 +1,284 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import sinon from "sinon";
+import { PropertyRecord, PropertyValueFormat } from "@itwin/appui-abstract";
+import { KeySet } from "@itwin/presentation-common";
+import { PresentationLabelsProvider, PresentationPropertyDataProvider } from "@itwin/presentation-components";
+import { getByRole as getByRoleRTL, render, waitFor } from "@testing-library/react";
+import userEvents from "@testing-library/user-event";
+import { MultiElementPropertyGrid, PropertyGridManager } from "../../property-grid-react";
+import { createPropertyRecord, stubFavoriteProperties, stubPresentation, stubSelectionManager } from "../TestUtils";
+
+import type { IModelConnection } from "@itwin/core-frontend";
+import type { InstanceKey } from "@itwin/presentation-common";
+import type { ISelectionProvider, SelectionChangeEventArgs } from "@itwin/presentation-frontend";
+
+describe("<MultiElementPropertGrid />", () => {
+  const imodel = {} as IModelConnection;
+
+  let getDataStub: sinon.SinonStub<Parameters<PresentationPropertyDataProvider["getData"]>, ReturnType<PresentationPropertyDataProvider["getData"]>>;
+  let getLabelsStub: sinon.SinonStub<Parameters<PresentationLabelsProvider["getLabels"]>, ReturnType<PresentationLabelsProvider["getLabels"]>>;
+  let keysStub: sinon.SinonStub<any[], any>;
+  let selectionManager: ReturnType<typeof stubSelectionManager>;
+
+  before(() => {
+    getDataStub = sinon.stub(PresentationPropertyDataProvider.prototype, "getData");
+    getLabelsStub = sinon.stub(PresentationLabelsProvider.prototype, "getLabels");
+    keysStub = sinon.stub(PresentationPropertyDataProvider.prototype, "keys");
+    sinon.stub(PropertyGridManager, "translate").callsFake((key) => key);
+
+    selectionManager = stubSelectionManager();
+    stubFavoriteProperties();
+    stubPresentation();
+  });
+
+  after(() => {
+    sinon.restore();
+  });
+
+  afterEach(() => {
+    keysStub.reset();
+    getDataStub.reset();
+    getLabelsStub.reset();
+  });
+
+  it("renders single instance properties", async () => {
+    setupMultiInstanceData([{
+      key: { id: "0x1", className: "TestClass" },
+      value: "Test-Value",
+    }]);
+
+    const { getByText } = render(
+      <MultiElementPropertyGrid
+        imodel={imodel}
+      />
+    );
+
+    await waitFor(() => getByText("Test-Value"));
```

Comment:
should we check if the button to open element list is **not** rendered (similar but opposite to the test below)?

Reply:
Added check.